client/client_update.go

```diff
@@ -351,3 +352,41 @@ func makeUpdateFetchRequest(url string) (*http.Request, error) {
 	}
 	return req, nil
 }
+
+// GetUpdateControlMap - requests an udpate control map refresh from the server
+func GetUpdateControlMap(
+	api ApiRequester,
+	serverURL,
+	deploymentID string,
+) (cm *updatecontrolmap.UpdateControlMap, err error) {
+	ep := fmt.Sprintf("/device/deployments/%s/update_control_map", deploymentID)
+	requestURL := buildApiURL(serverURL, ep)
+	request, err := http.NewRequest(
+		http.MethodGet,
+		requestURL,
+		nil,
+	)
+	if err != nil {
+		return nil, err
+	}
+	response, err := api.Do(request)
+	if err != nil {
+		return nil, err
+	}
+	defer response.Body.Close()
+	switch response.StatusCode {
+	case http.StatusOK:
+		responseBody, err := ioutil.ReadAll(response.Body)
+		if err != nil {
+			return nil, err
+		}
+		cm = new(updatecontrolmap.UpdateControlMap)
```

Comment:
response body is not simply update control map, it's an object containing update control map under "update_control_map" key
```
{
    "update_control_map": update_control_map
}

```
example from our docs:
```
{
  "update_control_map": {
    "id": "e1ba5bee-c45d-11eb-887d-3fd47bdc3990",
    "priority": 1,
    "states": {
      "ArtifactInstall_Enter": {
        "action": "pause"
      }
    }
  }
}

```